db_release_scripts/mosip_ida/sql/1.2.0.1_to_1.2.1_upgrade.sql

```diff
@@ -38,3 +38,13 @@ VALUES('IDA_KYC_EXCHANGE', 1095, true, 'mosipadmin', now(), NULL, NULL, false, N
 
 INSERT INTO ida.key_policy_def (app_id, key_validity_duration, is_active, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes, pre_expire_days, access_allowed) 
 VALUES('IDA_KEY_BINDING', 1095, true, 'mosipadmin', now(), NULL, NULL, false, NULL, 60, 'NA');
+
+INSERT INTO ida.partner_data(partner_id, partner_name, certificate_data, partner_status, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES ('mpartner-default-esignet', 'mpartner-default-esignet','94d4be61-31f0-42db-97be-8f4953f41fc6' , 'Active', 'superadmin', '2020-12-16 12:30:13.973', NULL, NULL, false, NULL);
+
+INSERT INTO ida.policy_data(policy_id, policy_data, policy_name, policy_status, policy_description, policy_commence_on, policy_expires_on, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+VALUES ('mpolicy-default-esignet', '{"trustBindedAuthVerificationToken":true,"allowAuthRequestDelegation":true,"allowKycRequestDelegation":true}', 'mpolicy-default-esignet', 'ACTIVE', 'mpolicy-default-esignet', '2023-12-16 12:30:14.183', '2028-04-28 09:37:00.000', 'superadmin', '2020-12-16 12:30:14.100', NULL, NULL, false, NULL);
+
+INSERT INTO ida.partner_mapping(partner_id, policy_id, api_key_id, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
```

Comment:
can you check if its required to add entry in partner_mapping table? As we have policy_id column in misp_license_data table.

Reply:
updated the same anusha